ext/mysqli/mysqli.stub.php

```diff
@@ -113,7 +113,7 @@ public function get_server_info() {}
     public function get_warnings() {}
 
     /**
-     * @return mysqli|false
+     * @return mysqli|null|false
```

Comment:
This method can only return null or false. It can't return mysqli object.

Reply:
Oh, nice! I didn't check if mysqli is a possible return type.

Reply:
I've just fixed this commit up

Reply:
Thanks. FYI, `mysqli_init()` can return `mysqli|false` but `mysqli::init()` can return `null|false`